plugins/chat/spec/system/page_objects/chat/components/thread_header.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Components
+    module Chat
+      class ThreadHeader < PageObjects::Components::Base
+        attr_reader :context
+
+        SELECTOR = ".chat-thread-header"
+
+        def initialize(context)
+          @context = context
+        end
+
+        def has_content?(content)
+          find(context).find(SELECTOR).has_content?(content)
```

Comment:
what I do now is:

```ruby
def component
  find(context)
end
```

Reply:
Yeah saw that in your other PR, it's nice, will update